.github/workflows/ci.yml

```diff
@@ -60,6 +60,12 @@ jobs:
             python-version: 3.8
             omp: n
             mpi: y
+          - python-version: 3.9
+            omp: n
+            mpi: n
+          - python-version: '3.10'
+            omp: n
+            mpi: n
```

Comment:
Instead of adding a special job for 3.10, let's use the latest version 3.10 in the build matrix above (line 28) and then have special cases for 3.7, 3.8, and 3.9.

Reply:
certainly, I have had a go at this. If I've understood correctly this effectively makes py3.10 the main test version instead of py3.8.